mvc-mvp/src/com/baeldung/mvc/Product.java

```diff
@@ -0,0 +1,45 @@
+package com.baeldung.mvc;
+
+public class Product {
+    private String productName;
+    private String productDescription;
+    private Double productPrice;
```

Comment:
Simplify these -- name, description, price -- and refactor the getters/setters accordingly